deps/llvm-3.7.1_tools.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/cmake/modules/AddLLVM.cmake b/cmake/modules/AddLLVM.cmake
+index 45f6746..dc1f37b 100644
+--- a/cmake/modules/AddLLVM.cmake
++++ b/cmake/modules/AddLLVM.cmake
+@@ -613,13 +613,16 @@ macro(add_llvm_tool name)
+     set(EXCLUDE_FROM_ALL ON)
+   endif()
+   add_llvm_executable(${name} ${ARGN})
++  if( NOT DEFINED LLVM_TOOLS_INSTALL_DIR )
```

Comment:
can llvm use the standard names for this instead of inventing our own? (https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html)